styles.css

```diff
@@ -0,0 +1,116 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.top-header {
+  display: flex;
+  position: fixed;
+  justify-content: space-between;
+  width: 100%;
+  height: 72px;
+  left: 0;
+  top: 0;
+  z-index: 1;
+  background-color: #3c3a39;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  width: 100%;
+}
+
+.humburger {
+  padding-left: 205px;
+  padding-top: 20px;
+}
+
+.logo-art {
+  padding-top: 15px;
+  width: 100%;
+  height: 24px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  padding-left: 16px;
+}
+
+#main {
+  background-color: #1c1a19;
+  color: white;
+  background-image: url(images/background.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  background-size: 44%;
+  padding: 120px 24px;
+  margin-top: 70px;
+}
+
+h1 {
+  width: 100%;
+  height: 124px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+h2 {
+  width: 100%;
+  height: 27px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  margin-top: 20px;
+  padding-top: 50px;
+}
+
+p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  align-content: center;
+  width: 100%;
+  height: 123px;
+  margin-top: 20px;
+}
+
+.social-media {
+  list-style: none;
+  padding: 7px 0 3%;
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  margin-top: 30px;
+}
```

Comment:
Nice work on the icons 💯 they look perfect but their alignment and spacing could use some work, look to increase the `gap` between these elements. This is what you have so far 👇🏽 

![Screenshot 2023-03-22 at 05 30 57](https://user-images.githubusercontent.com/45739963/226860262-64836787-d822-43c5-8dfb-14abca421852.png)

This is what Figma is expecting. 👇🏽 

![Screenshot 2023-03-22 at 05 31 11](https://user-images.githubusercontent.com/45739963/226860368-35b9a8af-e49d-414b-8218-66de64268517.png)

Reply:
hi [didierganthier](https://github.com/didierganthier) 
working on this can you please sugest on how to fix this alinment

Reply:
Hi @techmoves increasing the `gap` doesn't fix it? 🤔

Reply:
hi   [didierganthier](https://github.com/didierganthier)

yeah gap increased now

Reply:
hi   [didierganthier](https://github.com/didierganthier)
it should be better now

Reply:
hi [didierganthier](https://github.com/didierganthier)

please approve if need to be i think it okay now